### DISCUSSION TITLE
How to contribute to this project?

### DISCUSSION THREAD

**Discussion by yaten2302** [teeotsa/windows-11-debloat#32] [Q&A] [answered]
Hi @teeotsa, 

I want to contribute to this project, but I don't know much about PowerShell scripting. Could you please tell that from where it should be learnt so as to contribute to this project?

**Comment by teeotsa** [comment, marked as answer]
Hey!

Thank you for reaching out, I appreciate your effort. Good luck on your journey!

https://learn.microsoft.com/en-us/training/modules/introduction-to-powershell/
https://learn.microsoft.com/en-us/powershell/
https://www.tutorialspoint.com/powershell/index.htm
